### DISCUSSION TITLE
Treating videos as time spent on computer

### DISCUSSION THREAD

**Discussion by Lhotka** [rcaelers/workrave#546] [Ideas]
As of this moment, workrave doesn't treat video play (such as YouTube) as time spent on your device. Could this be possible?

**Comment by jjramsey** [comment]
I'm not clear about what you're asking. Do you want Workrave to treat time spent playing video as time that you are *not* idle, even if your hands are away from the keyboard and mouse?

**Reply by Lhotka** [reply]
Yes, since you're still sitting on your chair / in front of your screen.

**Comment by pieterdd** [comment]
Have you considered using Reading Mode? https://workrave.org/docs/menus/reading-mode/

> The timers will keep running until the first rest break, even if you stop using the keyboard and mouse. During this time you will be reminded to take microbreaks. After the first restbreak, the break timers stop until you start using keyboard and mouse again. Then it all starts over.
